packages/runtime/package.json

```diff
@@ -11,7 +11,8 @@
     "manifest.yml"
   ],
   "dependencies": {
-    "@netlify/esbuild": "0.14.39",
+    "@netlify/blobs": "^1.6.1",
+    "@netlify/esbuild": "0.14.39-1",
```

Comment:
TODO: THis fork is already deprecated we should use esbuild directly! (follow up PR)